docs/tutorials/extending_admin_ui/5_creating_a_udw_tab.md

```diff
@@ -0,0 +1,536 @@
+
+Next, add the following configuration blocks to `src/EzSystems/ExtendingTutorialBundle/Resources/config/services.yml`:
+
+``` yaml
+ezplatform.udw.image.panel.module.js:
+    parent: EzSystems\EzPlatformAdminUi\Component\ScriptComponent
+    arguments:
+        $src: '/bundles/acmetutorial/js/modules/ImagesPanel.module.js'
+    tags:
+        - { name: ezplatform.admin_ui.component, group: 'custom-admin-ui-modules' }
+
+ezplatform.udw.add.tab.js:
+    parent: EzSystems\EzPlatformAdminUi\Component\ScriptComponent
+    arguments:
+        $src: '/bundles/acmetutorial/js/add.tab.js'
+    tags:
+        - { name: ezplatform.admin_ui.component, group: 'custom-admin-ui-config' }
+```
+
+Finally, make sure that the configuration is imported. In `src/EzSystems/ExtendingTutorialBundle/DependencyInjection` add the following files:
+
+#### `Configuration.php`
+
+``` php
+<?php
+
+namespace EzSystems\ExtendingTutorialBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files.
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/configuration.html}
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('ez_systems_udw_tab_tutorial');
+
+        // Here you should define the parameters that are allowed to
+        // configure your bundle. See the documentation linked above for
+        // more information on that topic.
+
+        return $treeBuilder;
+    }
+}
+```
+
+#### `EzSystemsExtendingTutorialExtension.php`
+
+``` php
+<?php
+
+namespace EzSystems\ExtendingTutorialBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\DependencyInjection\Loader;
+
+/**
+ * This is the class that loads and manages your bundle configuration.
+ *
+ * @link http://symfony.com/doc/current/cookbook/bundles/extension.html
+ */
+class EzSystemsExtendingTutorialExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration();
+        $config = $this->processConfiguration($configuration, $configs);
+
+        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('services.yml');
+    }
+}
+```
+
+## Check results
+
+!!! tip
+
+    If you cannot see the results, clear the cache and reload the application.
+
+At this point you can go to the Back Office and choose "Browse" under "Content/Content structure".
```

Comment:
choose **Browse** under **Content/Structure**

Reply:
Content/Content structure is the actual order of things you need to click (when starting in the Dashboard, for example)